test/unit/core/requests.ts

```diff
@@ -1,14 +1,39 @@
 import { Request } from 'groupby-api';
 import * as sinon from 'sinon';
+import Autocomplete from '../../../src/core/adapters/autocomplete';
 import ConfigAdapter from '../../../src/core/adapters/configuration';
 import PastPurchaseAdapter from '../../../src/core/adapters/pastPurchases';
 import PersonalizationAdapter from '../../../src/core/adapters/personalization';
 import SearchAdapter, { MAX_RECORDS } from '../../../src/core/adapters/search';
 import Requests from '../../../src/core/requests';
 import Selectors from '../../../src/core/selectors';
+import * as utils from '../../../src/core/utils';
 import suite from '../_suite';
 
-suite('requests', ({ expect, stub }) => {
+suite('requests', ({ expect, stub, spy }) => {
+  describe('override', () => {
+    it('should return a function that calls override function with given request and pastRequest key', () => {
```

Comment:
Consider "... that calls the given override function..." to make it flow better.